_posts/2021-03-04-working-with-uswds-team.md

```diff
@@ -0,0 +1,52 @@
+excerpt: Learn how the Accelerator team collaborated with other digital teams and end-users across the federal government to bring new and updated components to the USWDS community in Release 2.10.
+---
+
+[Release 2.10]({{ site.baseurl }}/about/releases/#version-2100) of the U.S. Web Design System (USWDS) included eight contributions to the design system from a separate product team in GSA’s Technology Transformation Services. We focused on providing tools and guidance to help support federal agencies’ emergency response efforts.
+
+Our team is fortunate to include folks who’ve worked with the USWDS product team on several [10x](https://10x.gsa.gov/) projects, as well as designers and developers who’ve used the design system at other agencies. Lessons learned from two recent 10x projects — [10x Advanced Form Components](https://trello.com/c/r9uQ20rP/75-uswds-advanced-form-controls-formerly-digital-forms-experience-pack) and [10x Accessibility in Data Visualizations](https://trello.com/c/OOOres7a/63-improving-accessibility-in-data-visualizations-formerly-uswds-visualization-tool) — really helped us get an understanding of how work flows into the design system so we could dive in and get early traction.
+
+We identified three key things that contributed to our team success in working with the USWDS product team on projects over the past year.
+
+### 1. Two, three, four (or more!) minds are better than one
+
+Within the “guard rails” established in our [team agreement](https://tech.gsa.gov/guides/agile_team_working_agreement/) at kickoff, our project team self-organized to great effect. We worked in concert with the core USWDS product team to establish a sprint cadence, daily retros, weekly check-ins with the Product Owner, and biweekly check-ins with the core team. Ultimately, we also came up with schedules and structures that fit the work—staying collaborative without compromising efficiency. Outside of established team ceremonies, team members synced up in daily user experience (UX) standups, twice-weekly engineering touch bases, cross-pollination meetings, and pairing and workshopping sessions as needed.
+
+This approach allowed UX and engineering pairs to work on specific components in parallel to get more done, mobbing with additional team members when tackling thorny issues while preserving the heads-down time needed for high quality work.
+
+### 2. Ask for feedback early and often
+
+Understanding the impact our decisions have on end-users of the design system as well as public users of the websites built using USWDS was critical to keeping us on the right path.
+
+We [interviewed digital teams across the federal government]({{ site.baseurl }}/whats-new/updates/2021/02/26/improving-emergency-response/), and got feedback from the product team, peers, and the [USWDS community]({{ site.baseurl }}/about/community/) to ensure we benefited from a diverse set of opinions, backgrounds, and experiences and reflecting that back into our work.
```

Comment:
Before publishing, please update this URL with the path to accelerator blog post #1